dotcom-rendering/src/components/Dateline.tsx

```diff
@@ -25,12 +26,30 @@ const hoverUnderline = css`
 `;
 
 // for liveblog smaller breakpoints article meta is located in the same
-// container as standfirst and needs the same styling as standfirst
-const standfirstColouring = css`
-	${until.desktop} {
-		color: ${palette('--standfirst-text')};
+// container as standfirst and needs the same styling as standfirst on web
+const liveblogStyles = ({
+	isLiveBlog,
+	isApps,
+}: {
+	isLiveBlog: boolean;
+	isApps: boolean;
+}) => {
```

Comment:
The recommended way to compose emotion style is to use an array that is passed to the `css` prop. Would it be possible to keep it this way here?

```tsx
const liveBlogStyles = css`
	${until.desktop} {
		color: var(--mobile-colour);
	}
`;

const Component = ({ isApps } : { isApps: boolean }) => (
	<div
		css={liveBlogStyles}
		style={{ "--mobile-colour": isApps ? palette('--dateline-mobile') : palette('--standfirst-text')}}
	>Dateline</div>
);
```